<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entity View Tester</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item>
      <widget class="simQt::EntityTreeComposite" name="entityTreeComposite">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="platformButton">
       <property name="text">
        <string>Add Platform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="beamButton">
       <property name="text">
        <string>Add Beam</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gateButton">
       <property name="text">
        <string>Add Gate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>Add 10000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="toolTip">
        <string>Delete the selected entity(s)</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>simQt::EntityTreeComposite</class>
   <extends>QWidget</extends>
   <header>simQt/EntityTreeComposite.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <connections/>
</ui>
